src/components/SlateEditor/plugins/table/__tests__/combined-test.ts

```diff
@@ -19,7 +19,7 @@ const editor = withHistory(withReact(withPlugins(createEditor(), learningResourc
 describe("combined table plugin tests", () => {
   test("id in th and td is preserved on serialize and normalize", () => {
     const html =
-      '<section><table><tbody><tr><th scope="row" id="r0" data-align="right"><p>1</p></th><td id="abc" data-align="right"><p>2</p></td></tr></tbody></table></section>';
```

Comment:
Hvorfor fjerner du id her? Er ikke dette akkurat det tilfellet som denne PR'en skal fikse?

Reply:
Bare jeg som ikke leste test beskrivelsen. Må jo ha thead og table rows for at denne skal være riktig. Oppdatert nuh